manifest/content.xml

```diff
@@ -73,4 +73,27 @@
     <button />
   </form>
 
+  <!--
+    Tabs
+      Required childr elements: options, option, tab-content, tab
+      Each option element requires a single content:text child element
+      The number of tab elements should be the same of the number of option element
+  -->
+  <tabs>
+    <tab>
+      <label>
+        <text i18n-id="{{uuid}}">Option A</text>
```

Comment:
it is namespaced, the default XML namespace on this document is the content namespace :)